loki_role/tasks/grafana.yaml

```diff
@@ -0,0 +1,63 @@
+---
+- name: Install prerequisite packages on Debian-based systems
+  ansible.builtin.apt:
+    name:
+      - gnupg
+      - curl
+    state: present
+  when: ansible_facts['os_family'] == 'Debian'
+
+- name: Add Grafana GPG key on Debian-based systems
+  ansible.builtin.apt_key:
+    url: https://packages.grafana.com/gpg.key
+    state: present
+  when: ansible_facts['os_family'] == 'Debian'
+
+- name: Add Grafana repository on Debian-based systems
+  ansible.builtin.apt_repository:
+    repo: "deb https://packages.grafana.com/oss/deb stable main"
+    state: present
+  when: ansible_facts['os_family'] == 'Debian'
+
+- name: Update apt package repository on Debian-based systems
```

Comment:
can't we do update cache and install grafana in single. Command I think it is achievable please look into  this